rails-snow-queen/app/assets/javascripts/map.js

```diff
@@ -1,3 +1,10 @@
+constants = {
```

Comment:
this is now global, and I'm not sure it should be... at the very least, would have to be namespaced (what if you want to add other constants in the QuotePanel file? now your names will collide. I would put these either inside the contstructor, or define them after the class like:
```javascript
class Map {
  // ...
} 
Map.ZOOM = 15
Map.CENTER_MAP_LOCATION = { ... }
```

Reply:
it was an accident, have now declared constants as a const